stacks-signer/src/cli.rs

```diff
@@ -126,6 +126,10 @@ pub struct SignArgs {
     /// Path to config file
     #[arg(long, value_name = "FILE")]
     pub config: PathBuf,
+    /// The reward cycle the signer is registered for and wants to sign for
+    /// Note: this must be the current reward cycle of the node
+    #[arg(long, short)]
+    pub reward_cycle: u64,
```

Comment:
I'm curious why this is a CLI argument?  Is there a reason the signer can't just learn the upcoming reward cycle from `/v2/pox`?

Reply:
Ah this was a leftover change from when I thought you could vote for the current OR upcoming reward cycle. Then I updated the comment but didn't htink beyond that whether it was needed. Will remove.